lib/squasher.rb

```diff
@@ -18,7 +18,7 @@ def clean
 
   def rake(command, description = nil)
     tell(description) if description
-    system("bundle exec rake #{ command }")
+    system("bundle exec rake #{ command } 2>/dev/null")
```

Comment:
In some environment, we sometimes use same databases in spit of using multiple datatabases.

If there are duplicated database names in config/database.yml, an exception will be raised;"unkown database error" after drop 1st database or 
"database already exists" before create 2nd database.

So, drop these exceptions.